README.md

```diff
@@ -31,33 +46,41 @@ getting started, or view the full [Conduit roadmap][roadmap].
 
 1. Install the Conduit CLI with `curl https://run.conduit.io/install | sh `.
 
-2. Add `$HOME/.conduit/bin` to your `PATH`.
+1. Add `$HOME/.conduit/bin` to your `PATH`.
 
-3. Install Conduit into your Kubernetes cluster with:
-  `conduit install | kubectl apply -f -`.
+1. Install Conduit into your Kubernetes cluster with `conduit install | kubectl apply -f -`.
 
-4. Verify that the installation succeeded with `conduit check`.
+1. Verify that the installation succeeded with `conduit check`.
 
-5. Explore the Conduit controller with `conduit dashboard`.
+1. Explore the Conduit controller with `conduit dashboard`.
 
-6. Optionally, install a [demo application][conduit-demo] to run with Conduit.
+1. Optionally, install a [demo application][conduit-demo] to run with Conduit.
 
-7. Add [your own service][conduit-inject] to the Conduit mesh!
+1. Add [your own service][conduit-inject] to the Conduit mesh!
 
 ## Working in this repo ##
 
 [`BUILD.md`](BUILD.md) includes general information on how to work in this repo.
 
+We :heart: pull requests! See [CONTRIBUTING.md](CONTRIBUTING.md) for info on
+contributing changes.
+
+## Dependencies ##
+
+There are some projects used by Linkerd2 that are not part of this repo.
+
+* [linkerd2-proxy][proxy] -- High-performance data plane, injected as a sidecar with
+  every service.
+* [linkerd2-proxy-api][proxy-api] -- gRPC API bindings for the proxy
```

Comment:
Can you either add a period at the end of this bullet point, or remove the period from the end of the previous bullet point? I just want them to be consistent.